files/nagios/nrpe_fail2ban.te

```diff
@@ -0,0 +1,36 @@
+
+module nrpe_fail2ban 1.0;
+
+require {
+	type admin_home_t;
+	type nrpe_t;
+	class unix_dgram_socket sendto;
+	class file execute;
+	class file getattr;
+	class file { read getattr open };
+	class file execute_no_trans;
+	type fail2ban_client_exec_t;
+	class file { ioctl getattr };
+	class file { read open };
+	class file execute_no_trans;
+	type fail2ban_var_run_t;
+	class sock_file write;
+	class file ioctl;
+	type fail2ban_t;
+	class unix_stream_socket connectto;
+}
+
+#============= nrpe_t ==============
+allow nrpe_t admin_home_t:file execute;
+allow nrpe_t self:unix_dgram_socket sendto;
+allow nrpe_t admin_home_t:file getattr;
+allow nrpe_t admin_home_t:file { read open };
+allow nrpe_t admin_home_t:file execute_no_trans;
+allow nrpe_t admin_home_t:file ioctl;
+allow nrpe_t fail2ban_client_exec_t:file getattr;
+allow nrpe_t fail2ban_client_exec_t:file execute;
+allow nrpe_t fail2ban_client_exec_t:file { read open };
+allow nrpe_t fail2ban_client_exec_t:file execute_no_trans;
```

Comment:
On my EL7 system, I have to following SELinux contexts:
```
# seinfo -t | grep -i fail2ban
   fail2ban_var_lib_t
   fail2ban_var_run_t
   fail2ban_client_t
   fail2ban_tmp_t
   fail2ban_exec_t
   fail2ban_client_exec_t
   fail2ban_t
   fail2ban_log_t
   fail2ban_initrc_exec_t
```
After resetting the SELinux label, it works without the additional SELinux module:
```
# ls -lZ /usr/lib64/nagios/plugins/check_fail2ban 
-rwxr-xr-x. root root unconfined_u:object_r:nagios_unconfined_plugin_exec_t:s0 /usr/lib64/nagios/plugins/check_fail2ban
```
So, sorry for confusing you. :wink: